tests/utils/test_utils.py

```diff
@@ -420,13 +420,8 @@ def new_init(self, **kwargs):
             yield
 
 
-def check_is_stream(dataset: IDataset):
-    if type(dataset) == Dataset:
-        assert dataset.is_stream == False
-    elif type(dataset) == StreamDataset:
-        assert dataset.is_stream == True
-    else:
-        raise ValueError(type(dataset))
+def check_is_stream(dataset: IDataset, stream: bool):
```

Comment:
Light question: why did you separate the type comparator in test codes directly?

Reply:
This comes just from the momentum. The former one was kind of complicate one so I implemented it as the separate function.